src/data_processing/wiki_to_netflix.py

```diff
@@ -115,14 +114,17 @@ def process_data(test=False):
     missing_count = 0
     processed_data = []
 
-    netflix_data = read_netflix_txt(os.path.join(data_dir, 'movie_titles.txt'), test)
-    num_rows = len(netflix_data)
+    netflix_file = read_netflix_txt(os.path.join(data_dir, 'movie_titles.txt'), test)
```

Comment:
It looks like you accidentally renamed this to`netflix_file`. It should be `netflix_data` right?

Reply:
Yes I believe you are correct 😅 Fixed